src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -381,6 +389,10 @@ public function filter_add_body_classes( $classes ) {
 	 * @return string
 	 */
 	public function filter_template_file( $file, $name, $template ) {
+		if ( is_404() ) {
```

Comment:
Don't need this one to be `$query->is_404()` as well?

Reply:
Not on this case because we have the global WP Query there already.